signingscript/docker.d/init_worker.sh

```diff
@@ -261,6 +261,8 @@ case $ENV in
         test_var_set 'AUTOGRAPH_AUTHENTICODE_EV_USERNAME'
         test_var_set 'AUTOGRAPH_MAR_RELEASE_PASSWORD'
         test_var_set 'AUTOGRAPH_MAR_RELEASE_USERNAME'
+        test_var_set 'AUTOGRAPH_STAGE_XPI_PASSWORD'
+        test_var_set 'AUTOGRAPH_STAGE_XPI_USERNAME'
```

Comment:
I think these should be in dev/fake-prod, not prod?

Reply:
Sigh, thanks.. Good to see the test caught it at least!